app/resources/linked_account_resource.rb

```diff
@@ -0,0 +1,17 @@
+class LinkedAccountResource < BaseResource
+  model_hint model: LinkedAccount::MyAnimeList
+
+  attribute :token
+  attributes :external_user_id, :share_to, :share_from, :sync_to
+  # :type is reserved for STI
```

Comment:
For ListImport we do `alias_attribute :kind, :type` in the model, and then expose `kind` as an attribute here.  Also can you combine the two lines here?  There's no reason to have 'em separate